internal/server/v1/triples/golden/bulk_triples_in/result1.json

```diff
@@ -4,7 +4,7 @@
       "node": "Country",
       "triples": {
         "rangeIncludes": {
-          "entities": [
+          "nodes": [
```

Comment:
To confirm, this API is not used by anyone?

Reply:
Nope, this is not used.